app/models/concerns/queue_it/queable.rb

```diff
@@ -31,32 +35,32 @@ def get_next_in_queue
       end
     end
 
-    def formatted_queue(node_attribute)
+    def formatted_queue(nodable_attribute)
       return if local_queue.empty?
 
       if local_queue.one_node?
-        [local_queue.head_node.nodable.send(node_attribute)]
+        [local_queue.head_node.nodable.send(nodable_attribute)]
       elsif local_queue.two_nodes?
-        [local_queue.head_node.nodable.send(node_attribute) +
-          local_queue.tail_node.nodable.send(node_attribute)]
+        [local_queue.head_node.nodable.send(nodable_attribute),
+          local_queue.tail_node.nodable.send(nodable_attribute)]
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/ArrayAlignment: Align the elements of an array literal if they span more than one line.